src/store/account/accountSlice.ts

```diff
@@ -1,14 +1,7 @@
-import { createSlice, createAsyncThunk, PayloadAction } from '@reduxjs/toolkit'
```

Comment:
accountSlice has become a bit generic. We could probably take this further and separate out notifications. I'm wondering if we can remove `hotspots` now that we have `hotspotsSlice`?

Reply:
yeah I was thinking we could work towards that. just didn't want to bite off too much here